nemo_text_processing/inverse_text_normalization/zh/clean_eval_data.py

```diff
@@ -0,0 +1,342 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
```

Comment:
this file is copied from english folder and has not been adapted. either delete or adjust for chinese